homeassistant/components/velbus/const.py

```diff
@@ -1,3 +1,6 @@
 """Const for Velbus."""
 
+ATTR_MEMO_TEXT = "memo_text"
```

Comment:
Please rename this to follow the naming conventions of schema keys.
```suggestion
CONF_MEMO_TEXT = "memo_text"
```

Reply:
Fixed in https://github.com/home-assistant/core/pull/31222/commits/aa7594916cc5f01452dde4fc72971c0b8bb2a2d8